lib/rule-tester/rule-tester.js

```diff
@@ -272,11 +272,49 @@ class RuleTester {
         this.rules[name] = rule;
     }
 
+    /**
+     * @typedef {{
+     *   code: string,
+     *   options?: any,
+     *   filename?: string,
+     *   parserOptions?: Linter.ParserOptions,
+     *   settings?: { [name: string]: any },
+     *   parser?: string,
+     *   globals?: { [name: string]: boolean }
```

Comment:
```suggestion
     *   globals?: { [name: string]: "readonly" | "writable" }
```

Reply:
I look up the doc and make it `"readonly" | "writable"| "off"`.